src/sentry/testutils/cases.py

```diff
@@ -1567,3 +1577,76 @@ def setUp(self):
         )
         self.name = self.user.get_display_name()
         self.short_id = self.group.qualified_short_id
+
+
+@apply_feature_flag_on_cls("organizations:metrics")
+@pytest.mark.usefixtures("reset_snuba")
+class MetricsAPIBaseTestCase(SessionMetricsTestCase, APITestCase):
+    ...
+
+
+class OrganizationMetricMetaIntegrationTestCase(MetricsAPIBaseTestCase):
+    def setUp(self):
+        self.login_as(user=self.user)
```

Comment:
Do we need to call `super().setUp()` here?

Reply:
yep missed that